app/forms/eligibility_questions.py

```diff
@@ -0,0 +1,49 @@
+from flask import render_template
+from flask_wtf import FlaskForm
+from wtforms import IntegerField
+from wtforms.validators import DataRequired
+from wtforms.validators import NumberRange
+
+
+def minimium_money_question_page(
+    max_amount: int, success_url: str, data_hook: callable = print
+):
+    """Returns a Flask view function which checks the eligibility
+    of a fund using the gov.uk design system.
+
+    Args:
+        max_amount (int): The maximium amount one can apply for.
+        success_url (str): The url one is redirected to if eliglible.
+        data_hook (callable): This function will be ran on the inputted
+        data if the fund is eligible. By default prints to console.
+
+    Returns:
+        function: A function which renders a template.
+    """
+
+    class MinimiumMoneyForm(FlaskForm):
+
+        money_field = IntegerField(
+            label="project_money_amount",
+            validators=[DataRequired(), NumberRange(max=max_amount)],
+        )
+
+    def min_money_page():
+
+        form = MinimiumMoneyForm()
+        if form.money_field.data is not None and not form.validate_on_submit():
```

Comment:
Could `if not form.money_field.data` be a pythonic alternative?

Reply:
Tried this and it didn't work. Will refactor this today to be more pythonic.

Reply:
How about only validating the form if the `if request.POST and form.validate_on_submit():`

Reply:
Hi Harry, 
Thanks for the insightful suggestion. I've added this change :)
Please re-review and let me know if you're happy with this PR.
J